### ISSUE TITLE
Example is broken

### ISSUE THREAD

**Issue by gsovereignty** [faassen/solid-dexie#3]
Looks like you need to import Dexie first or something?

All I get is `db is not defined`.

Do you have any projects that are using this library so I can see an example?

**Comment by paxee** [comment]
Yes @gsovereignty , you need to also install dexie and setup your db (see db-fixture.ts file), this is just a solidjs wrapper for creating reactive live queries.
